policies/design-process.md

```diff
@@ -1,160 +1,167 @@
+    or evolve the implementation, but also to contributors who wish to implement
+    related functionality or understand non-obvious rationales behind given
+    design decisions.
+
+It is recognised that designs will often necessarily change once implementation
+begins. In the majority of cases, the understanding of the problem domain will
+evolve and improve after implementation begins and this will indicate further
+changes to the design. This is an iterative process of progressive refinement.
+It is an explicit objective of this policy to support and encourage this
+agile-style process, as opposed to a waterfall-style process in which designs
+must be approved and finalised prior to implementation.
+
+This policy adopts a graded system of review in which the degree of formal
+approval by the OTC which a design must undergo is proportionate to the scope
+and risks of the design. For example, a design which involves new or evolved
+public APIs may require a greater amount of scrutiny, whereas a purely internal
+design may require that the OTC simply be notified of the design document and
+invited to comment, with implicit approval in the absence of objections.
+
+Requirement for Design Documents
+--------------------------------
+
+A design document is required for any proposed enhancement which adds new APIs
+or non-trivially evolves or modifies existing APIs.
+
+For any other kind of proposed enhancement, a design document should be created
+if it incorporates design decisions or aspects significant enough to warrant
+one. For example, if an enhancement adds a new internal module with clearly
+delineated boundaries with a documented internal API which can be consumed by
+other code internal to OpenSSL, a design document is desirable. This example is
+not exhaustive. The proposer may use their discretion in determining whether a
+design document is desirable, but any OTC member may require that a design
+document be produced.
+
+Levels of Scrutiny
+------------------
+
+There are three levels of scrutiny which can be applied to a design, listed
+below in ascending order of severity:
+
+ - Notify
+ - Present
+ - Approve
+
+These levels work as follows:
+
+ - At the **Notify** level, the OTC is notified of a new design
+   when it is available for review, via an email to the openssl-project mailing list.
+   OTC members and committers can review and comment on the design.
+   A minimum waiting time of one week applies to ensure OTC members have the
+   opportunity to review the design.
+
+   The notification does not need to be made by the author of the design
+   document.
+
+ - At the **Present** level, the OTC is notified of a new design
+   in the same way that it is at the Notify level. The same minimum waiting time
+   applies. The design is also introduced and explained in a presentation given
+   by its author to the OTC in a meeting of the OTC. The OTC has opportunities
```

Comment:
I don't think it necessarily has to be the author - although that would be normal. For example, in the case of an external contribution someone might present the design on the author's behalf (e.g. if the OTC meeting is in the middle of the night for the contributor)